docker-compose-ci.yml

```diff
@@ -43,7 +43,7 @@ services:
     volumes:
       - ./test/db/:/docker-entrypoint-initdb.d/
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.3.0
+    image: elasticsearch:6.4.3
```

Comment:
Was having issues with that `elastic.co` repo.  So switched to the official dockerhub instance, which is supposedly the same, but not hosted by Elastic.  Also upgraded to 6.4, which is what prod-cms uses right now.